42/README.md

```diff
@@ -206,7 +206,18 @@ With every consequent migration, the migration height is overwritten.
 
 Since BigchainDB retains the blocks built by old Tendermint chains, the HTTP API offers the exact same experience as if there were no migrations.
 
+#### Migration election specs
+
+We introduce two transaction specs, with operation types [`TENDERMINT_MIGRATION_ELECTION`][spec_tendermint_migration_election] and [`TENDERMINT_MIGRATION_ELECTION_VOTE`][spec_tendermint_migration_election_vote], for the purpose of implementing migration elections. `TENDERMINT_MIGRATION_ELECTION` is an extension of `CREATE`; `TENDERMINT_MIGRATION_ELECTION_VOTE` is an extension of `TRANSFER`.
```

Comment:
I wouldn't call those two things "specs." They're two new operation types, to be added to the existing BigchainDB Transactions Spec v2 (BEP-13).